charts/thoras/README.md

```diff
@@ -71,6 +73,8 @@ helm install \
 | thorasOperator.limits.memory | String | 1000Mi | Thoras Operator memory limit |
 | thorasOperator.requests.cpu | String | 1000m | Thoras Operator CPU request |
 | thorasOperator.requests.memory | String | 1000Mi | Thoras Operator memory request |
+| thorasOperator.slackWebhookUrl | String | "" | Slack Webhook URL destination for notifications |
```

Comment:
I think it's worth allowing these to be configured at the service level, but I also welcome feedback on whether this is making it more complicated than necessary.